docs/v2/v2.0.rst

```diff
@@ -12,8 +12,8 @@ used for storing Zarr arrays. The key words "MUST", "MUST NOT", "REQUIRED",
 Status
 ------
 
-This specification is the latest version. See :ref:`zarr-specs` for previous
-versions.
+This version is outdated and not actively maintained. See :ref:`zarr-specs` for
```

Comment:
Is 'This version is outdated and not actively maintained' good?

Reply:
I don't think we need to change this text here. let's keep the changes to the absolute minimum required to address the linked issue.

Reply:
I'm sorry, I don't understand.

Not changing the text to say, 'This specification is the latest version' in the V2 spec doesn't solve the issue.

Reply:
oops, I was confused about which file this was, sorry about that. I recommend saying "This specification has been superseded. See :ref:`zarr-specs` for the latest version.". I now see that it doesn't make sense to keep the original reference to previous versions, but just saying "there's a newer version" means nobody has to touch this text again if we release v4.

Reply:
'This specification has been superseded.' is a better statement.
Thanks, @d-v-b.